docs/manual/manual.md

```diff
@@ -228,7 +230,12 @@ Now your haxophone is ready to play!
    Blow air through the mouthpiece or tip of the neck to begin playing.
 
 4. **Stop Playing:**
```

Comment:
With the new section you are adding right below this, this section should probably be removed.
```suggestion
```